src/components/Input.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+function Input({ label, attribute, onChange }) {
+	return (
+		<div className='flex justify-center'>
+			<div className='mb-3 xl:w-96'>
+				<label
+					for='exampleFormControlInput1'
```

Comment:
The `for` attribute is `htmlFor` and this must be equal to the `id` of the input.